src/package_io/input_parser.star

```diff
@@ -135,6 +149,22 @@ def input_parser(plan, input_args):
 
 def parse_network_params(plan, input_args):
     results = {}
+
+    # configure interop
+
+    results["interop"] = default_interop_args()
+    results["interop"].update(
+        input_args.get("interop", {})
+    )
+    results["interop"]["supervisor_params"] = default_supervisor_params()
+    if "supervisor_params" in input_args["interop"]:
```

Comment:
I've updated the code to follow this pattern with some minor modifications, namely using `results` instead of `input_args` on the LFS of the assignment as that seems to mirror your intent more closely, please LMK if that's not correct. also, as `.update(...)` returns `None`, I split up the assignment and replacement